<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная по ОС №3</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Макс. цилиндров</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Нач. поз.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="cylinderCount"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="queryCount"/>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="startPosition"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Сгенерировать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="shuffleButton">
       <property name="text">
        <string>Перемешать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Запуск</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Запросов</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QComboBox" name="algoBox"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTableWidget" name="inputTable">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QTableWidget" name="outputTable">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QScrollBar" name="directionBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Направление</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="7">
      <widget class="HddWidget" name="hddSheme" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HddWidget</class>
   <extends>QWidget</extends>
   <header>hddwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
